tests/foreman/api/test_repository.py

```diff
@@ -303,62 +658,47 @@ def test_positive_create(self):
                     product=product,
                     url=DOCKER_REGISTRY_HUB,
                 ).create()
-                repo2 = repo.read()
-                self.assertEqual(repo.name, repo2.name)
+                self.assertEqual(repo.name, name)
                 self.assertEqual(
-                    repo.docker_upstream_name,
-                    repo2.docker_upstream_name
-                )
-                self.assertEqual(repo.content_type, repo2.content_type)
+                    repo.docker_upstream_name, u'busybox')
+                self.assertEqual(repo.content_type, u'docker')
 
     @tier2
     @run_only_on('sat')
-    @skip_if_bug_open('bugzilla', 1217603)
     def test_positive_synchronize(self):
```

Comment:
if this is still docker-only, i would reflect that in the name of the test test_positive_docker_synchronize ..or similar.